main.go

```diff
@@ -76,26 +77,29 @@ func main() {
 		}
 
 		contentLineSplit := bytes.Split(content, []byte{'\n'})
-		data = bytes.Split(contentLineSplit[0], delimiter)
 
-		lineLength := len(data)
-		var output [][]byte
+		if (bytes.Contains(contentLineSplit[0], delimiter) && *onlyDelimitParameter) || !*onlyDelimitParameter {
```

Comment:
Any way to shorten this condition?

Reply:
i dont tihnk so

Reply:
remove the brackets?

Reply:
Swap the two expressions of the ||. Maybe then you'll see the way I mean.

Reply:
still dont get it

Reply:
After the swapping the left and right side of the || remove the right side of the &&. That's what I mean.

Reply:
if (bytes.Contains(contentLineSplit[0], delimiter) && || *onlyDelimitParameter {
?????

Reply:
No, `if !*onlyDelimitParameter || bytes.Contains(contentLineSplit[0], delimiter) {`.